test/fixtures/3.0/ping/src/migrate.ts

```diff
@@ -0,0 +1,20 @@
+import {PingApplication} from './application';
+
+export async function migrate(args: string[]) {
```

Comment:
I don't see how is migration relevant for this example app, can you remove this file please?